dashboard/client/src/App.tsx

```diff
@@ -250,8 +254,8 @@ const App = () => {
                     />
                     <Route
                       element={
-                        <SideTabPage tabId="applications">
-                          <ServeApplicationsListPage />
+                        <SideTabPage tabId="deployments">
```

Comment:
what happened to applications? can they still be viewed?

Reply:
There's no application list page, but you can see the name of the application next to each deployment, and you can also use a filter to filter down to specific applications.

Someone can also still go to the application detail page by clicking into an application.

In general, people who use ray serve deal directly with deployments way more than applications.

Reply:
ok, maybe this should be in a release note on next ray release